AWSiOSSDKv2.xcodeproj/xcshareddata/xcschemes/AWSIoT.xcscheme

```diff
@@ -46,6 +46,11 @@
                BlueprintName = "AWSIoTTests"
                ReferencedContainer = "container:AWSiOSSDKv2.xcodeproj">
             </BuildableReference>
```

Comment:
Have we run these tests with Thread Sanitizer on to ensure no data races?

Reply:
Yes, I found and resolved a few minor issues identified by Thread Sanitizer. Once was a value called `ticks` which I handled with a Dispatch Queue.